### PR TITLE
Docs: document that `@time expr` returns the value of `expr`

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

File: base/timing.jl [hunk 1]
```diff
@@ -221,6 +221,8 @@ allocations, and the total number of bytes its execution caused to be allocated,
 returning the value of the expression. Any time spent garbage collecting (gc), compiling
```
Comment (giordano) [inline base/timing.jl:221]: Isn't it written already here?

Comment (DilumAluthge) [reply base/timing.jl:221]: 🤦

Comment (DilumAluthge) [reply base/timing.jl:221]: I swear I know how to read.

Comment (DilumAluthge) [reply base/timing.jl:221]: Not my best moment.

**Pull request closed without merging**